classes/local/object_manipulator/manipulator.php

```diff
@@ -25,6 +25,8 @@
 
 namespace tool_objectfs\local\object_manipulator;
 
+use dml_exception;
```

Comment:
This doesnt seem to be necessary. Mightve been auto pulled in by the IDE

Reply:
it just looks a bit cleaner with the `use` statement. :)